controlplane/src/utils/utils.ts

```diff
@@ -0,0 +1,10 @@
+import { CompositionOptions } from '../types/index.js';
```

Comment:
should we just put this in the other utils file(inside core)?

Reply:
ok